pulse-protobuf/proto/pulse/config/inflow/v1/k8s_prom.proto

```diff
@@ -73,7 +74,7 @@ message KubernetesPrometheusConfig {
   // https://prometheus.io/docs/prometheus/latest/http_sd/
   message HttpServiceDiscovery {
     // The URL to scrape for service discovery. This should be a valid URL.
-    string url = 1 [(validate.rules).string = {min_len: 1}];
+    common.v1.EnvOrInline url = 1 [(validate.rules).string = {min_len: 1}];
```

Comment:
This should have a message required validation `[(validate.rules).message = {required: true}]`